.env.example

```diff
@@ -15,6 +15,7 @@ VUE_APP_BUSINESS_REGISTRY_URL="https://dev.business-registry-dashboard.bcregistr
 VUE_APP_DOCUMENTS_UI_URL="https://dev.documents.bcregistry.gov.bc.ca/"
 VUE_APP_NOTICE_OF_WITHDRAWAL_FORM_URL="https://www2.gov.bc.ca/assets/gov/employment-business-and-economic-development/business-management/permits-licences-and-registration/registries-forms/form_19_-_notice_of_withdrawal.pdf"
 VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
+VUE_APP_PEOPLE_BUSINESS_REGISTRY_URL="https://dev.people.business.bcregistry.gov.bc.ca/en-CA/"
```

Comment:
I recommend calling this `PERSON_ROLES_URL` to keep consistent with the repo name and what's in 1Password and so it's not tied to whatever the URL value is.